entgql/template.go

```diff
@@ -172,6 +174,25 @@ type fieldCollection struct {
 	Mapping []string
 }
 
+// decap takes a string and decapitalizes the first letter
+//
+//	FullName  => fullName
+//	ID  => iD
+func decap(s string) string {
```

Comment:
The generated names are available here: https://github.com/ent/contrib/blob/master/entgql/schema.go#L578-L594. We should bind them to the template data when generating these types (or the <T>WhereInput) either statically or by template function.

Reply:
Sounds good, I will update.